sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/implementation/compatibility/AzureSpringBootVersionVerifier.java

```diff
@@ -16,16 +16,13 @@
 
 public class AzureSpringBootVersionVerifier {
     private static final Logger LOGGER = LoggerFactory.getLogger(AzureSpringBootVersionVerifier.class);
-    final Map<String, CompatibilityPredicate> supportedVersions = new HashMap<String, CompatibilityPredicate>() {
-        {
-            this.put("2.5", AzureSpringBootVersionVerifier.this.is2_5());
-            this.put("2.6", AzureSpringBootVersionVerifier.this.is2_6());
-        }
-    };
+    final Map<String, CompatibilityPredicate> supportedVersions = new HashMap<>();
```

Comment:
why this is not private?

Reply:
Yes, it looks not preferable.  we should refactor here. For now, this field is accessed out of this class so the modifier cannot be private directly. I can probe and refactor it.